scripts/custom_lint_checks.py

```diff
@@ -73,17 +74,77 @@ def visit_call(self, node):
         Args:
             node. Call. The current function call node.
         """
-        if (isinstance(node.func, astroid.Name) and
+        if (isinstance(node.func, astroid.Name)) and (
                 node.func.name == self._function_name):
-            if (node.args is not None and
+            if (node.args is not None) and (
                     len(node.args) > self._positional_arguments_count):
                 self.add_message('non-explicit-kwargs', node=node)
 
 
+class HangingIndentChecker(BaseChecker):
+    """Custom pylint checker which checks for break after paren in case
+    of hanging indentation.
+    """
+    __implements__ = IRawChecker
+
+    name = 'hanging-indent'
+    priority = -1
+    msgs = {
+        'C0002': (
+            (
+                'There should be a break after parenthesis when content within '
+                'paren spans to multiple lines.'),
+            'break-after-hanging-indent',
+            (
+                'If something within parenthesis extends along multiple lines, '
+                'break after opening paren.')
+        ),
+    }
+
+    def process_module(self, node):
+        """Process a module.
+
+        Args:
+            node: Node to access module content.
+        """
+        file_content = node.stream().readlines()
+        file_length = len(file_content)
+        exclude = False
+        for line_num in range(file_length):
+            line = file_content[line_num].lstrip().rstrip()
+            if line.startswith('"""') and not line.endswith('"""'):
+                exclude = True
+            if line.endswith('"""'):
+                exclude = False
+            if line.startswith('#') or exclude or '# pylint: disable' in line:
```

Comment:
Please note that disabling any pylint check will disable this check as well since all disabling is done by: `# pylint: disable <msg-name/msg-code>`. Therefore change this accordingly.

Reply:
I wrote this for tests in the previous style because some errors of form ``( some content #pylint: disable`` were also caught. Since the checks for now only look for a comma-separated list, this is not an issue now and I have removed it.

Reply:
I think this would come handy to us. Suppose we want to disable the checker for some line then we could simply do: `# pylint: disable: no-break-after-hanging-indent` for that line. Therefore it would be better if we retain this. Thanks!

Reply:
I think that can be done automatically by writing ``pylint: disable=hanging-indent`` since this is a plugin for pylint.

Reply:
Yes, this works (I tested it locally for both the custom checkers). Thanks!